src/registrar/assets/src/sass/_theme/_alerts.scss

```diff
@@ -24,13 +24,33 @@
   .usa-alert__body.margin-left-1 {
     margin-left: 0.5rem!important;
   }
+
+  .usa-alert__body--widescreen::before {
+    left: 4rem !important;
+  }
+  .usa-alert__body--widescreen {
+    padding-left: 7rem!important;
+  }
 }
 
-// NOTE: !important is used because _font.scss overrides this
-.usa-alert__body--widescreen {
-  max-width: $widescreen-max-width !important;
+/*----------------
+   Alert Fonts
+-----------------*/
+// Fixes some font size disparities with the Figma
+// for usa-alert alert elements
+.usa-alert {
+  .usa-alert__heading.larger-font-sizing {
+    font-size: units(3);
```

Comment:
I don't know why this is here.  Didn't write it, but am all for upgrading it.

Reply:
Looked around...the problem is .usa-alert__heading overrides these USWDS font classes.  I don't want to start modifying previous design ACs, but it is something I'll note because this tells me we have an inconsistency within our design.